### PR TITLE
Set max length of 10,000 characters for terraform statement evaluation

### PR DESCRIPTION
**Pull request by nimrodkor**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

The rendering can cause very, very long strings to get created. Most of them are not useful so it's a waste of time to try and evaluate them - this sets a configurable limit (with a friendly log message)

### CODE REVIEW

File: checkov/terraform/graph_builder/variable_rendering/renderer.py [hunk 1]
```diff
@@ -279,6 +283,12 @@ def evaluate_non_rendered_values(self) -> None:
             self.local_graph.update_vertex_config(vertex, changed_attributes)
 
     def evaluate_value(self, val: Any) -> Any:
+        val_length: int = len(str(val))
+        if CHECKOV_RENDER_MAX_LEN and 0 < CHECKOV_RENDER_MAX_LEN < val_length:
```
Comment (gruebel) [inline checkov/terraform/graph_builder/variable_rendering/renderer.py:287]: either
```python
if 0 < CHECKOV_RENDER_MAX_LEN < val_length:
```
or 
```python
if CHECKOV_RENDER_MAX_LEN and CHECKOV_RENDER_MAX_LEN < val_length
```
and then `0` is the value to render any length

**Review by gruebel — CHANGES_REQUESTED** [review]

File: checkov/terraform/graph_builder/variable_rendering/renderer.py (hunk 1 above)
Comment (nimrodkor) [reply checkov/terraform/graph_builder/variable_rendering/renderer.py:287]: First option is no good - CHECKOV_RENDER_MAX_LEN can be `None`. I'll use the second

Comment (gruebel) [reply checkov/terraform/graph_builder/variable_rendering/renderer.py:287]: ah ok, then how about
```python
if CHECKOV_RENDER_MAX_LEN is not None and 0 < CHECKOV_RENDER_MAX_LEN < val_length:
```

**Review by gruebel — APPROVED** [review]
🍻

**Pull request merged**